src/Core/Form/IdentifiableObject/DataHandler/EmployeeFormDataHandler.php

```diff
@@ -124,7 +124,8 @@ public function create(array $data)
             $data['language'],
             $data['active'],
             $data['profile'],
-            isset($data['shop_association']) ? $data['shop_association'] : $this->defaultShopAssociation
+            isset($data['shop_association']) ? $data['shop_association'] : $this->defaultShopAssociation,
+            $data['has_enabled_gravatar'] ?? false
```

Comment:
Is the `??` check necessary? You don't use it in the update function
And seems like both provider and Form always include this field, so it should always be present

Reply:
@jolelievre You are right.